streampark-console/streampark-console-service/src/main/resources/mapper/system/TeamMapper.xml

```diff
@@ -47,8 +47,8 @@
 
     <select id="findUserTeams" resultType="team" parameterType="team">
         select t.* from t_team t
-        join t_user_role ur
-        on t.id = ur.team_id
+        join t_member m
+        on t.id = m.team_id
         where ur.user_id = #{userId}
```

Comment:
`ur`should be changed to `m`. It's a bug, the `test_user3` cannot login platform.

Reply:
> `ur`should be changed to `m`. It's a bug, the `test_user3` cannot login platform.

There is actually no problem here.

Reply:
Checked, it's done.